src/google_servers.py

```diff
@@ -0,0 +1,8 @@
+GOOGLE_SERVERS = [
+    "khms1.google.com",
```

Comment:
can use "khm.google.com" and "mt.google.com"
They have load balance

Reply:
That's good to know. The question is whether the LB will round-robin or maintain a session connection. In which case there's it the risk of throttling again.